src/panels/lovelace/editor/view-editor/hui-view-editor.ts

```diff
@@ -130,26 +161,26 @@ export class HuiViewEditor extends LitElement {
     fireEvent(this, "view-config-changed", { config });
   }
 
-  private _computeLabel = (
-    schema: SchemaUnion<ReturnType<typeof this._schema>>
```

Comment:
Please don't undo this typing.  The `localize` keys are only passing the lint now because of the exceptions in `localize.ts` which I am removing one by one.